docs/source/gateway/index.rst

```diff
@@ -857,6 +879,8 @@ MLflow Models
 You can also build and deploy MLflow Models that call the MLflow AI Gateway.
 The example below demonstrates how to use an AI Gateway server from within a custom ``pyfunc`` model.
 
+For a full walkthrough and example of using the MLflow serving integration to query a model directly through the MLflow AI Gateway, please see `the example <https://github.com/mlflow/mlflow/tree/master/examples/gateway/mlflow_serving/README.md>`_.
```

Comment:
This example is different from the previous 2 sentences that talk about querying the gateway from an MLflow model. Can we move this into a separate section, e.g. split these into subsections with some text at the top that says "you can query the gateway from an MLflow Model and also query an MLflow Model from the gateway?"